src/StackAdmin/Azs.Compute.Admin/Module/Azs.Compute.Admin/Azs.Compute.Admin.psd1

```diff
@@ -56,14 +56,18 @@ Licensed under the MIT License. See License.txt in the project root for license
     # ProcessorArchitecture = ''
 
     # Modules that must be imported into the global environment prior to importing this module
-    RequiredModules      = @(@{ModuleName = 'AzureRM.Profile'; RequiredVersion = '3.4.1'; },
-                             @{ModuleName = 'AzureRM.Resources'; RequiredVersion = '4.4.1'; })
+    RequiredModules   = @(@{ModuleName = 'AzureRM.Profile'; ModuleVersion = '5.3.4'; },
```

Comment:
versions

Reply:
Good catch, I removed that but then when I merged all my Admin files were having merge conflicts.